### ISSUE TITLE
[A11y Review] Button

### ISSUE THREAD

**Issue by mendeza** [ITS-HCD/nysds#1096]
# [Button](https://designsystem.ny.gov/components/button/)

## Findings

- superfluous `tabindex="0"`
- superfluous `role="button"`
- captive `aria-label` redundantly echos label value; cannot be user specified.
    - should only be set: (A) when there is no text content, e.g., just an icon or arbitrary graphic; (B) when additional context is needed for ATs
